routes/web.php

```diff
@@ -1,7 +1,13 @@
 <?php
 
+
 use Illuminate\Support\Facades\Route;
+use App\Http\controllers\Admin\JobController;
```

Comment:
make C in `controllers` capital, this can cause bug

Reply:
done

Reply:
done with @esraa33-95 esraa